easy_infra.yml

```diff
@@ -74,6 +74,11 @@ packages:
     - all
     version: v2.1.8
     version_argument: --version
+  git:
+    helper:
+    - all
```

Comment:
This will ensure that `git --version` is run and successful in the correct images, due to this:

https://github.com/SeisoLLC/easy_infra/blob/971a571328b29a8254de1bf0f5f0b1e158508fc4/tests/test.py#L63-L149

One example [here](https://github.com/SeisoLLC/easy_infra/actions/runs/5958880313/job/16163772092#step:13:70)

But it lacks other useful tests of the logging end-to-end.

I want to forego that for the time being, because we'll be refactoring this all soon to move to otel, but can you please put this on the backlog and share the task link?  Specifically we need to add tests of easy_infra log generation.  This is one that we will likely implement using goss and start running the different testing frameworks in parallel